### PR TITLE
i264: rename "signing in with firebase.." loading text 

### LINKED ISSUE #264: rename "signing in with firebase.." loading text
#### Is your feature request related to a problem? Please describe.
After signing in with an auth provider, we see "signing in with firebase..." + loading widget.
This is really helpfull as a developer to see what step in the sign in process we are on, but as a user, the word firebase would confuse me.

#### Describe the solution you'd like
I would like to rename "signing in with firebase.." to "signing into crowdleague".

### PR DESCRIPTION
**Pull request by nickmeinhold**
fixes #264 

- I also changed the text on app initialization to avoid suspicious words like "redux" and "firebase" (sort of out of scope but maybe not sort of?)

### CODE REVIEW

File: lib/widgets/app/initializing_indicator.dart [hunk 1]
```diff
@@ -18,9 +18,9 @@ class InitializingIndicator extends StatelessWidget {
   Widget build(BuildContext context) {
     var message = '';
     if (!firebaseDone) {
-      message = 'Waiting for Firebase...';
+      message = 'Enticing a ghost into the machine...';
     } else if (!reduxDone) {
-      message = 'Waiting for Redux...';
+      message = 'Plumbing the pipes...';
```
Comment (tamari-gray) [inline lib/widgets/app/initializing_indicator.dart:23]: works as an abstract visualization for the user to understand whats going on and for us devs as a literal point to look at,
cause this redux structure has got some pipage

File: lib/widgets/auth/auth_page/auth_page.dart [hunk 2]
```diff
@@ -39,11 +39,11 @@ class _AuthPageState extends State<AuthPage> {
                 case AuthStep.signingInWithGoogle:
                   return WaitingIndicator('Contacting Google...');
                 case AuthStep.signingInWithEmail:
-                  return WaitingIndicator('Signing In With Email...');
+                  return WaitingIndicator('Signing In with Email...');
                 case AuthStep.signingUpWithEmail:
-                  return WaitingIndicator('Signing Up With Email...');
+                  return WaitingIndicator('Creating your Account...');
```
Comment (tamari-gray) [inline lib/widgets/auth/auth_page/auth_page.dart:44]: ooh i feel special now (as a user)

**Review by tamari-gray — APPROVED** [review]
LTRB 👍

File: lib/widgets/app/initializing_indicator.dart (hunk 1 above)
Comment (nickmeinhold) [reply lib/widgets/app/initializing_indicator.dart:23]: Awesome, that' s what I was hoping

Comment (nickmeinhold) [reply lib/widgets/app/initializing_indicator.dart:23]: Thanks!

File: lib/widgets/auth/auth_page/auth_page.dart (hunk 2 above)
Comment (nickmeinhold) [reply lib/widgets/auth/auth_page/auth_page.dart:44]: Haha YOUR account... I actually had it as Creating an Account but had a last minute thought :-)

Comment (tamari-gray) [reply lib/widgets/auth/auth_page/auth_page.dart:44]: noice haha

**Pull request merged**